### PR TITLE
fix special handing load data local sql keep putting SQL on the queue inner 971

### PR DESCRIPTION
**Pull request by ylinzhu**
Reason:  
  BUG  
Type:  
  BUG 
Influences：  
   fix xx

### CODE REVIEW

File: src/main/java/com/actiontech/dble/services/mysqlsharding/LoadDataProtoHandlerImpl.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    @Override
+    public ProtoHandlerResult handle(ByteBuffer dataBuffer, int dataBufferOffset, boolean isSupportCompress) {
+        ProtoHandlerResult.ProtoHandlerResultBuilder resultBuilder = handlerResultBuilder(dataBuffer, dataBufferOffset, isSupportCompress);
+        ProtoHandlerResult result = resultBuilder.build();
+        switch (result.getCode()) {
+            case COMPLETE_PACKET:
+                byte[] packetData = result.getPacketData();
+                if (packetData != null) {
+                    if (isEndOfDataFile(packetData)) {
+                        loadDataHandler.end(packetData[3]);
```
Comment (dcy10000) [inline src/main/java/com/actiontech/dble/services/mysqlsharding/LoadDataProtoHandlerImpl.java:30]: reset protoHandler in here maybe a good idea

File: src/main/java/com/actiontech/dble/server/handler/ServerLoadDataInfileHandler.java [hunk 2]
```diff
@@ -888,6 +889,7 @@ private String ignoreLinesDelete(String strSql) {
 
 
     public void clear() {
+        service.getConnection().setProto(new MySQLProtoHandlerImpl());
```
Comment (dcy10000) [inline src/main/java/com/actiontech/dble/server/handler/ServerLoadDataInfileHandler.java:892]: should keep this status in new "ProtoHandler" .

**Review by dcy10000 — COMMENTED** [review]
。

File: src/main/java/com/actiontech/dble/server/handler/ServerLoadDataInfileHandler.java (hunk 2 above)
Comment (ylinzhu) [reply src/main/java/com/actiontech/dble/server/handler/ServerLoadDataInfileHandler.java]: ok

**Review by dcy10000 — APPROVED** [review]
lgtm

**Pull request merged**